src/workerd/io/io-context.c++

```diff
@@ -1152,6 +1152,12 @@ auto IoContext::tryGetWeakRefForCurrent() -> kj::Maybe<kj::Own<WeakRef>> {
   }
 }
 
+static constexpr auto scriptWillNeverGenerateResponseErr =
+    "The script will never generate a response. Request response relies on "
+    "a Promise that will never be resolved during the Request's lifetime. "
+    "For more information go to "
+    "https://developers.cloudflare.com/workers/errors/script-will-never-generate-a-response";
```

Comment:
The URL currently returns a 404. We should likely hold off landing until that page is present.

Reply:
Yeah definitely - I PRed the docs and the redirect here: https://github.com/cloudflare/cloudflare-docs/pull/15014 - was planning to wait until that is in